src/marginals/Header.tsx

```diff
@@ -0,0 +1,39 @@
+import { Link } from "react-router-dom";
+import { SlBag, SlUser } from "react-icons/sl";
+
+function Header() {
+  return (
+    <nav className="navbar navbar-expand-md bg-light sticky-top px-5">
+      <div className="container-fluid">
+        <div className="d-flex col-4">
+          <Link className="nav-link mx-3" to="/shop">
```

Comment:
React router should have a `NavLink` you can use to optionally style these links with an active state when you're on the corresponding page.